src/Compilers/Core/CodeAnalysisTest/Collections/ImmutableSegmentedDictionaryTest.cs

```diff
@@ -295,8 +295,9 @@ public void Indexer_KeyNotFoundException_ContainsKeyInMessage()
         {
             var map = ImmutableSegmentedDictionary.Create<string, string>()
                 .Add("a", "1").Add("b", "2");
-            var exception = Assert.Throws<KeyNotFoundException>(() => map["c"]);
-            Assert.Contains("'c'", exception.Message);
+            var missingKey = "__ThisKeyDoesNotExist__";
+            var exception = Assert.Throws<KeyNotFoundException>(() => map[missingKey]);
+            Assert.Contains(missingKey, exception.Message);
```

Comment:
wait what?  how is 'c' found in the map on a different locale?

Reply:
`c` wasn't found in the different locale. The Assert.Contains was looking for `'c'` but in the translation it was `"c"` to fit the language.

Reply:
ah, i see.  it was the `Contains` part, not hte indexing part.  phew... that makes much more sense.  Thanks!